index.html

```diff
@@ -5,6 +5,8 @@
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
 
+    <link rel="preconnect" href="https://techcrunch.com/wp-json/wp/v2/posts" />
```

Comment:
тут кстати указывается просто домен или поддомен
```suggestion
    <link rel="preconnect" href="https://techcrunch.com" />
```